scripts/validate-new-version.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+# Constants
+LABEL_BREAKING_CHANGE="breaking-change"
+
+# Check if the planned version is passed as an argument
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <new_planned_version>"
+  exit 1
+fi
+
+# Input: new planned version
+PLANNED_VERSION="$1"
+
+# Get the latest release information
+LATEST_RELEASE=$(gh release view --json tagName,publishedAt -q '{tag: .tagName, date: .publishedAt}')
+
+# Extract the tag and date from the latest release JSON
+TAG=$(echo "$LATEST_RELEASE" | jq -r .tag)
+TAG_DATE=$(echo "$LATEST_RELEASE" | jq -r .date)
+
+# Check if the latest release information was retrieved successfully
+if [ "$TAG" == "null" ]; then
+  echo "No releases found."
+  exit 1
+fi
+
+# Parse the latest tag for versioning
+IFS='.' read -r MAJOR MINOR PATCH <<< "$TAG"
+IFS='.' read -r P_MAJOR P_MINOR P_PATCH <<< "$PLANNED_VERSION"
+
+# List PRs merged after the latest release date and filter by breaking change label
```

Comment:
Only PRs to develop should be in this list I think?

Reply:
Good catch, I will modify it accordingly.

Reply:
Fixed this one.